scripts/no_lost_declarations.sh

```diff
@@ -104,25 +104,28 @@ else
   grep '\(+\|-\)' | sed 's=^ *1 =`=; s=^[^`]=`=; s=$=`='
 fi)"
 
+printf '*Diff of declarations*\n\n'
+
```

Comment:
Should the entire report also be inside a `<details>` if it is longer than X lines? (X = 5?)